resources/views/chops/home.blade.php

```diff
@@ -1,30 +1,21 @@
-@extends('app')
+@extends('app') @section('title') all chops @endsection
 
-@section('title')
-    all chops
-@endsection
 
+@section('content') @foreach($chops as $chop)
+<div>
+  <div>
+    <h2>{{$chop->chops_name}}</h2>
+  </div>
 
-@section('content')
-        @foreach($chops as $chop)
-            <div>
-                <div><h2>{{$chop->chops_name}}</h2></div>
+  @foreach($chop->uploads as $image)
 
-                @foreach($chop->uploads as $image)
+  <div>
+    <img src="{{$image->file_uri}}" alt="chops image"
+      class="image-rounded">
+  </div>
+  @endforeach
+  <p>{{$chop->about}}</p>
```

Comment:
space needed between `{{` and `$chop->about` . Check all occurrences of this kind and put the needed space